### PR TITLE
New package: WatchGuard.MobileVPNwithSSLclient version 12.10

### LINKED ISSUE #123941: [Package Request]: WatchGuard SSL VPN
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: 
Publisher: WatchGuard
Package Name: Mobile VPN with SSL v12.10 for Windows
Description: WatchGuard Mobile VPN client with SSL v12.10 for Windows
Package Version: 12.10
Installer URL: https://cdn.watchguard.com/SoftwareCenter/Files/MUVPN_SSL/12_10/WG-MVPN-SSL_12_10.exe

### PR DESCRIPTION
**Pull request by BrandonWanHuanSheng**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

Resolves #123941 

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/124169)

### CODE REVIEW

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml:4]:
```suggestion
PackageIdentifier: WatchGuard.MobileVPNWithSSLClient
```
PackageId is generally written in "Pascal" case

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: WatchGuard.MobileVPNWithSSLClient
```
PackageId is generally written in "Pascal" case

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.yaml:4]:
```suggestion
PackageIdentifier: WatchGuard.MobileVPNWithSSLClient
```
PackageId is generally written in "Pascal" case

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: "12.10"
+PackageLocale: en-US
+Publisher: WatchGuard
+PublisherUrl: https://www.watchguard.com/
+PublisherSupportUrl: https://www.watchguard.com/support
+PrivacyUrl: https://www.watchguard.com/privacy
+PackageName: Mobile VPN with SSL client
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:11]:
```suggestion
PackageName: WatchGuard Mobile VPN with SSL client
```
To match Apps&Features/Control Panel

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+License: WatchGuard API License
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:12]:
```suggestion
License: Proprietary
```
This doesn't look like a valid license type

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: "12.10"
+InstallerType: inno
+Installers:
+- Architecture: x86
+  InstallerUrl: https://cdn.watchguard.com/SoftwareCenter/Files/MUVPN_SSL/12_10/WG-MVPN-SSL_12_10.exe
+  InstallerSha256: B399888F952C327A35E2216BD3549E173496D1A6D781697713D508312F0A8131
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml:10]:
```suggestion
  InstallerSha256: B399888F952C327A35E2216BD3549E173496D1A6D781697713D508312F0A8131
  ProductCode: WatchGuard Mobile VPN with SSL client
```

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml [hunk 7]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: "12.10"
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.installer.yaml:5]:
```suggestion
PackageVersion: "12.10"
Scope: machine
```

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+LicenseUrl: https://www.watchguard.com/wgrd-trust-center/watchguard-apis-license-agreement
+Copyright: Copyright (c) WatchGuard Technologies, Inc.
+CopyrightUrl: https://www.watchguard.com/
+ShortDescription: Mobile VPN with SSL client Setup
+Moniker: MobileVPN
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:17]:
```suggestion
Moniker: mobilevpn
```
Moniker should be written in lowercase

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+Publisher: WatchGuard
+PublisherUrl: https://www.watchguard.com/
+PublisherSupportUrl: https://www.watchguard.com/support
+PrivacyUrl: https://www.watchguard.com/privacy
+PackageName: Mobile VPN with SSL client
+License: WatchGuard API License
+LicenseUrl: https://www.watchguard.com/wgrd-trust-center/watchguard-apis-license-agreement
+Copyright: Copyright (c) WatchGuard Technologies, Inc.
+CopyrightUrl: https://www.watchguard.com/
+ShortDescription: Mobile VPN with SSL client Setup
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:16]:
```suggestion
ShortDescription: The WatchGuard Mobile VPN with SSL client is a software application that is installed on a remote computer. The client makes a secure connection from the remote computer to your protected network through an unsecured network, such as the Internet. 
```

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+PackageIdentifier: WatchGuard.MobileVPNwithSSLclient
+PackageVersion: "12.10"
+PackageLocale: en-US
+Publisher: WatchGuard
+PublisherUrl: https://www.watchguard.com/
+PublisherSupportUrl: https://www.watchguard.com/support
+PrivacyUrl: https://www.watchguard.com/privacy
+PackageName: Mobile VPN with SSL client
+License: WatchGuard API License
+LicenseUrl: https://www.watchguard.com/wgrd-trust-center/watchguard-apis-license-agreement
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:13]:
```suggestion
# LicenseUrl:
```
This URL is about API license, I don't see a mention of the client

File: manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+Copyright: Copyright (c) WatchGuard Technologies, Inc.
+CopyrightUrl: https://www.watchguard.com/
```
Comment (mdanish-kh) [inline manifests/w/WatchGuard/MobileVPNwithSSLclient/12.10/WatchGuard.MobileVPNwithSSLclient.locale.en-US.yaml:15]:
```suggestion
# CopyrightUrl:
```
This is the PublisherUrl not the CopyrightUrl for the client

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to modify the directory structure to match the new PackageIdentifier case. The `.yaml` file names would also need to be modified

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by BrandonWanHuanSheng** [comment]
It seem like this one required my computer to modified.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**